app/containers/Forms/NumberFields.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import NumberFormat from 'react-number-format';
+import {FieldProps} from 'react-jsonschema-form';
+
+const NumberFields: React.FunctionComponent<FieldProps> = ({
+  formData,
+  onChange
+}) => {
+  return (
+    <NumberFormat
+      thousandSeparator
+      className="form-control"
+      allowNegative={false}
```

Comment:
This should depend on the `schema` prop. More precisely the `minimum` or `exclusiveMinimum` values, if defined:
```js
const allowNegative = (schema.minimum === undefined && schema.exclusiveMinimum === undefined) || minimum < 0 || exclusiveMinimum < 0
``` 
See https://json-schema.org/understanding-json-schema/reference/numeric.html

Reply:
Updated.